src/utils/sdkService.js

```diff
@@ -95,6 +95,15 @@ class SdkService {
 
     return { did, credentials }
   }
+  async getSharableCredentialLink() {
```

Comment:
This method suppose to recieve credentialId or credential, and its suppose to be credential, on which you click to share (NOT just `credentials[0].id`)

Reply:
made changes as you said